code/mob/living/carbon/human/procs/damage.dm

```diff
@@ -641,3 +641,40 @@
 /mob/living/carbon/human/UpdateDamage()
 	..()
 	src.hud?.update_health_indicator()
+
+///The deliberate act of using one's body to cover a live time-fused hand grenade.
+/mob/living/carbon/human/proc/cover_explosion_with_body(power, brisance=1)
```

Comment:
a decent amount of this proc appears to be copy-paste from `ex_act()` - any reason not to just call that to handle non-organ stuff?